### PR TITLE
Add dictionary input for setting aflr4AIM options.

### PR DESCRIPTION
**Pull request by bburke38**
Similar to recent updates in FUNtoFEM, this adds the ability to use dictionaries directly to set aflr4AIM options.

### CODE REVIEW

File: tacs/caps2tacs/aflr_aim.py [hunk 1]
```diff
@@ -51,6 +52,29 @@ def set_mesh(
...
+        Call this before setting up the TACS model. The dictionary should take
+        the form of, e.g.: 
+
+        dictOptions['aflr4AIM']['myOption'] = myValue
+        """
+        self._dictOptions = dictOptions
+
+        return self
+    
+    def _setDictOptions(self):
```
Comment (sean-engelstad) [inline tacs/caps2tacs/aflr_aim.py:68]: Can you make this _set_dict_options

**Review by sean-engelstad — APPROVED** [review]

**Comment by sean-engelstad** [comment]
@bburke38 can you add this `save_dict_options` in for the `TacsAim` class too for consistency?

**Comment by timryanb** [comment]
@bburke38 can we use this new verbosity flag to silence the caps print outs in the integration tests [here]([test_caps_shape_derivatives.py](https://github.com/smdogroup/tacs/blob/master/tests/integration_tests/test_caps_shape_derivatives.py)) and [here]([test_caps_thick_derivatives.py](https://github.com/smdogroup/tacs/blob/master/tests/integration_tests/test_caps_thick_derivatives.py))?

**Review by gjkennedy — APPROVED** [review]

**Comment by bburke38** [comment]
@timryanb Yes, we should be able to use the verbosity flag to silence those print outs. Do you want me to make a new pull request just for those?

**Comment by timryanb** [comment]
@bburke38 yes, that would be great

**Pull request merged**